### ISSUE TITLE
Questionable specification of _ID_BenefitSurtax_Switch policy parameter

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/Tax-Calculator#647]
In the ```current_law_policy.json``` file on the master branch, there is the following parameter specification:
```  
  "_ID_BenefitSurtax_Switch": {
        "long_name": "Deductions subject to the surtax on itemized deduction benefits",
        "description": "The surtax on itemized deduction benefits applies to the benefits derived from the itemized deductions specified with this parameter. ",
        "irs_ref": "",
        "notes": "",
        "start_year": 2013,
        "col_var": "MARS",
        "row_var": "FLPDYR",
        "row_label": ["2013"],
        "cpi_inflated": true,
        "col_label": ["Medical", "State & Local", "Real Estate", "Casualty", "Miscellaneous", "Interest", "Charity"],
        "value": [[true, true, true, true, true, true, true]]
    },
```

What sense does it make to inflation index boolean values?  Surely this should say:
```
"cpi_inflated": false,
```

Or am I missing something?

@MattHJensen @feenberg @Amy-Xu

**Comment by martinholmer** [comment]
After further reflection, another question arises.  What is the meaning of ```"col_var": "MARS"```?

That can't be correct, can it?  How about ``` "col_var": ""```, which is what is used for the EITC parameters that are indexed by the number of EITC-eligible children in the filing unit.

@MattHJensen @feenberg @Amy-Xu

**Comment by Amy-Xu** [comment]
I'll open an PR to fix this in a sec. Thanks @martinholmer

**Comment by martinholmer** [comment]
Issue #647 resolved by pull request #648.
